Userland/Applications/FileManager/DirectoryView.cpp

```diff
@@ -545,7 +545,11 @@ void DirectoryView::do_delete(bool should_confirm)
 
 bool DirectoryView::can_modify_current_selection()
 {
-    return !current_view().selection().is_empty() && access(path().characters(), W_OK) == 0;
```

Comment:
Lowercase "the" in this commit; and what does column refer to?

Reply:
oh it’s about the columns from the Columns View in the File Manager.

![image](https://user-images.githubusercontent.com/16520278/187967916-8758311d-15ba-4191-b319-0164b21770bd.png)

you couldn’t remove files from your home directory, because it checked permissions for the one column on the left.

Reply:
But the model itself is not inherently tied to any visual representation, there's also a table view and a grid view - please word that differently :)